docker-compose.yml

```
@@ -34,6 +34,7 @@ services:
       KMS_SESSION_CMK_ALIAS: 'alias/viewer-sessions-cmk-alias'
 
       # Local only
+      API_SERVICE_HOSTNAME: http://api-web
```

```suggestion
      API_SERVICE_HOSTNAME: api-web
```
The AWS variable won't have the protocol, se we need to remove it from here. And we'll need to add it somewhere in the PHP. Perhaps in the `ApiClient\Client`?

---

As discussed, we have moved the protocol to the terraform variable and renamed to URL